Documentation/dev/libvirt-howto.md

```diff
@@ -74,45 +83,7 @@ On Debian based distros, modify `/etc/default/libvirtd` and set:
 libvirtd_opts="--listen"
 ```
 
-Next, restart libvirt: `systemctl restart libvirtd`
-
-Finally, if you have a firewall, you may have to allow connections from the IP
```

Comment:
Where's this going?  Don't we still need to open up the firewall to let these connections through?

Reply:
We shouldn't need this any of this if we're deriving the libvirt URL from the cluster network. Rules will already be added by libvirt to allow traffic to the bridge / gateway from the nodes.